### DISCUSSION TITLE
Is there an active fork for adding nodes to ASTImporter?

### DISCUSSION THREAD

**Discussion by sztomi** [Ericsson/codechecker#3903] [Q&A] [answered]
I found a some forks linked from [here](https://fuchsia.googlesource.com/docs/+/refs/heads/sandbox/jakehehrlich/objcopy/ctu_analysis.md) but they are pretty outdated. I'd like to add some missing visitors, but I'd like to avoid duplicating effort. Is there an active fork somewhere aimed at improving the ASTImporter?

**Comment by bruntib** [comment, marked as answer]
Hi,
We are contributing directly to the official LLVM repo. The fork indicated in this linked page is really outdated and it is a personal fork.
